search-app/src/cmr/search/api/concepts_search.clj

```diff
@@ -160,7 +160,7 @@
         scroll-id-and-search-params (core-api/get-scroll-id-and-search-params-from-cache ctx short-scroll-id)
         scroll-id (:scroll-id scroll-id-and-search-params)
         cached-search-params (:search-params scroll-id-and-search-params)
-        ctx (assoc ctx :query-string body :scroll-id scroll-id)
+        ctx (assoc ctx :query-string body :scroll-id scroll-id :params params)
```

Comment:
We should not stick all the params into the context just for the purpose of getting `facets_size`. The `facets_size` info should be put into the query and passed through the query record to where it is being used.

Reply:
I couldn't find the good spot to put in the query. Could you point out where do you have in mind?

Reply:
Take a look at line 346 of `cmr.search.services.parameters.conversion`, `common-params/parse-query-level-params` muti-method processes the params and construct a map of query-attributes to put special attributes needed in query, e.g. `:result-features`.

Reply:
Done. Filed a ticket for the existing issue.